test/test.js

```diff
@@ -458,6 +460,27 @@ function checkRefTestResults(browser, id, results) {
   });
 }
 
+function browserTestReportHandler(req, res) {
+  var parsedUrl = url.parse(req.url, true);
+  var pathname = parsedUrl.pathname;
+  if (pathname === '/browserTestReports') {
+    // let information = req.body.browserTestInfo;
+    // var i = 0;
+    // ++i;
+    const writableStream = fs.createWriteStream(
```

Comment:
Use `var` instead of `const` since `const` is ES6 and this file is writen with ES5.